namui-cli/src/procedures/dev_wasm_electron.rs

```diff
@@ -0,0 +1,79 @@
+use crate::{
```

Comment:
There are many overlapping code with `dev_wasm_web`.
We can combine these procedures if necessary.

Reply:
Don't combine procedures, just combine services. If you combine procedures, it makes them un-flexible.